notebooks/clesto.ipynb

```diff
@@ -27,7 +27,7 @@
   },
```

Comment:
It's important that you "clear output" in the cell option of a notebook before committing to git